docs/kubectl-config-view.md

```diff
@@ -29,8 +29,10 @@ $ kubectl config view -o template --template='{{range .users}}{{ if eq .name "e2
 ### Options
 
 ```
+      --flatten=false: flatten the resulting kubeconfig file into self contained output (useful for making secrets)
   -h, --help=false: help for view
       --merge=true: merge together the full hierarchy of kubeconfig files
+      --minify=false: remove all information not used by current-context from the output (useful for making secrets)
```

Comment:
Agree its useful for secrets, but I am not sure if alluding to it in help text says how.  I would just leave the ( ... ) out of the help